language/stdlib/modules/libra_system.move

```diff
@@ -44,26 +43,32 @@ module LibraSystem {
         move_to_sender(CapabilityHolder { cap })
     }
 
-    // This returns a copy of the current validator set.
-    public fun get_validator_set(): T {
-        LibraConfig::get<T>()
-    }
-
     // This copies the vector of validators into the LibraConfig's resource
     // under ValidatorSet address
     fun set_validator_set(value: T) acquires CapabilityHolder {
         LibraConfig::set_with_capability<T>(&borrow_global<CapabilityHolder>(LibraConfig::default_config_address()).cap, value)
     }
 
+    ///////////////////////////////////////////////////////////////////////////
+    // Methods operating the Validator Set config
+    // callable by Validator's operators
+    ///////////////////////////////////////////////////////////////////////////
+
+    // Adds a new validator, this validator should met the validity conditions
```

Comment:
I guess in the same script we perform the rotation we can call this? can we make this idempotent? so we can keep calling it without causing an assert error? or maybe we should have two scripts rotate and add and just rotate?

Reply:
Which script do you mean by "the same script we perform the rotation" ?

Reply:
Good question! A transaction script

Reply:
Good question... Do you mean rotate_config or rotate_operator?

Reply:
depends on whether or not you have a list of potential validators. if you do, we would likely just perform that operation here. If you don't, then after rotate_config and you have a valid config.